src/main/webapp/WEB-INF/views/addDriversToCar.jsp

```diff
@@ -0,0 +1,13 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add Drivers to Car</title>
```

Comment:
We add single driver, not list, so make sense no call as plural (same for file name)